localization/localization-spanish/common-errors.md

```diff
@@ -0,0 +1,301 @@
+
+## Evite hacer traducciones absolutamente literales
+
+Las traducciones literales nunca son recomendadas. Es muy común querer traducir palabra por palabra la forma verbal del inglés para simular el flujo del idioma original. El español tiene una estructura diferente y utiliza modismos y estructuras verbales de manera distinta que el inglés.
+
+La sugerencia de esta guía es siempre leer la frase completa primero y entender el contexto detrás de ella para luego traducir. Trate de ver el todo de la frase y no sus palabras individuales para enviar el mismo mensaje, así no resulte necesariamente en una traducción literal.
+
+Es recomendable evitar _transliterar_ o _traducir palabra a palabra_, es decir, leer la primera palabra, traducirla, ir a la siguiente, traducirla, etcétera. Esto suele crear frases extrañas con una composición que no se siente natural al idioma español.
+
+### Ejemplo
+
+Consideremos esta frase:
+
+> This site uses cookies for incremental improvements. You may find the services function without them but at a reduced usability.
+
+La traducción literal tanto en palabras como gramatical resultaria en:
+
+> Este sitio usa _cookies_ para mejoras incrementales. Usted puede encontrar que los servicios funcionen sin ellas pero a una reducida usabilidad.
+
+Pero esto no se siente natural, tanto en vocablos como en gramática. Una traducción centrada más en la comprensión general de la frase sería algo como lo siguiente:
+
+> Este sitio utiliza _cookies_ para realizar mejoras continuas. Puede utilizar los servicios sin ellas pero con una usabilidad reducida.
+
+## Evite traducciones automáticas con traductores
+
+Es bastante común caer en la tentación de utilizar servicios de traducción como [el traductor de Google](https://translate.google.com) para obtener una traducción rápida.
+
+En principio, no tenemos objeciones sobre el uso de traductores para una base inicial, pero les suplicamos a las personas que hagan esto que revise _dos, tres o cuatro veces la traducción proporcionada por el traductor_.
+
+Muchas veces estos traductores hacen _traducciones literales_. Esto significa que deciden traducir términos de manera literal, sin conocer el contexto del texto o su nivel técnico.
+
+Por ejemplo, muchas veces se traducen palabras de nombres en inglés sin posibles traducciones propias al término técnico como __ (en el contexto de instalaciones externas)  o _cookies_ (en el contexto web), que no tendrían cabida en la traducción final.
+
+O peor aún, a veces ofrecen traducciones que es posible no tengan cabida en lo que se refiere. Por ejemplo, _sideload_ puede llegar a traducirse como «transferencia local» que, aunque técnicamente correcto, en el contexto de elementary no sería el término preciso y puede llegar a confundir en el contexto de elementary (en elementary estamos usando «instalación externa» por ejemplo)
```

Comment:
I wonder how this is translated in macOS? Maybe there's a precedent for this term there?

Reply:
Uhh, yeah, sideload. Added a longer explanation on this particular word on a comment below.